### PR TITLE
Remove Travis, add Circle, remove html files & modify origami.json

### PR DESCRIPTION
**Pull request by j-mes**
- Check for correct support link
- Change from Travis to Circle CI
- Move variables from /src/demos/config.js into origami.json
- Remove html files from /src/demos/
- Remove travis.yml
- Add circle.yml
- Passed OBT verify check
- Passed OBT server check

@AlbertoElias could you please check this branch with OBT --runServer? It complained about some Sass errors and would not run. I'm not sure whenever I should remove/correct those errors or not.

FYI: After investigation with @onishiweb, there was a map being put inside a string which caused errors with linting and in turn preventing from the demos being run. Changed from `#{$o-grid-gutters}` to `5px` in place instead.

### CODE REVIEW

**Comment by j-mes** [comment]
See updated pull request comment with Sass errors in `resized.scss`

**Comment by kaelig** [comment]
I believe the test demo is the only one that needs to load the JS (style-switcher).

File: origami.json [hunk 1]
```diff
@@ -1,44 +1,65 @@
...
+		{
+			"name": "ie8",
+			"template": "demos/src/layout.mustache",
+			"js": "demos/src/js/style-switcher.js",
+			"sass": "demos/src/scss/ie8.scss",
+			"expanded": false,
+			"description": "Forced IE 8 experience, where the width of the grid is fixed to be a M layout"
+		},
+		{
+			"name": "test",
```
Comment (kaelig) [inline origami.json]: Is this demo working properly?

According to the old config it should have these things in it:

```
demos.push({
	name: 'test',
	description: 'test demo',
	sass: 'demos/src/scss/default.scss',
	template: 'demos/src/test.mustache',
	data: 'demos/src/configurations.json',
	js: 'demos/src/js/style-switcher.js',
	documentClasses: 'test',
	expanded: false,
	dependencies: ['o-buttons@^2.0.0']
});
```

Comment (j-mes) [inline origami.json]: I'll be investigating this today. Thanks for pointing this out @kaelig!

**Comment by j-mes** [comment]
This has been amended and I have tried the test.html to see if it would switch between all the versions of the o-grid.

I have asked @AlbertoElias to give this a once-over to make sure all is good.

File: demos/src/scss/resized.scss [hunk 2]
```diff
@@ -9,5 +9,5 @@ $o-grid-gutters: (default: 5px);
 @import 'common';
 
 .test body:after {
-	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: #{$o-grid-gutters}';
+	content: '$o-grid-layouts: S: 400px, M: 800px, L: 1000px, $o-grid-gutters: 5px';
```
Comment (AlbertoElias) [inline demos/src/scss/resized.scss]: why does this need to change?

Comment (onishiweb) [inline demos/src/scss/resized.scss]: Because Sass was throwing an error. The variable was a Map, and that can't be output as a string.

Comment (j-mes) [inline demos/src/scss/resized.scss]: It failed linting, as a map could not be placed inside a string.

Comment (AlbertoElias) [inline demos/src/scss/resized.scss]: Wow, that's new. What about using map-get to get the correct value from the gutter map?

Comment (onishiweb) [inline demos/src/scss/resized.scss]: Possibly. Although as this is just for the demo (and also not a bit of code that's likely to be "copy-pasted"), I don't think there's a huge issue in it being hardcoded.

Comment (AlbertoElias) [inline demos/src/scss/resized.scss]: But if gutters are modified, it will break. If we already have one source of truth, why not use it?

Comment (onishiweb) [inline demos/src/scss/resized.scss]: I'm not sure I follow. How will it break? And how are the breakpoint values being hardcoded ok, but the gutter not?

Comment (AlbertoElias) [inline demos/src/scss/resized.scss]: yeah, probably none of them should be hardcoded, but it's not that important. We can leave it as is. Feel free to merge

**Pull request merged**